src/app/app.test.tsx

```diff
@@ -1,13 +1,11 @@
 import * as React from 'react';
-import { render, waitFor, screen } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
 import { App } from '.';
-import { APP_TITLE } from './common/constants';
 
 describe('App', () => {
   test('renders welcome page without errors', async () => {
     render(<App />);
-    await waitFor(() => screen.getByRole('heading'));
-    expect(screen.getByRole('heading')).toHaveTextContent(APP_TITLE);
+    expect(screen.getByRole('heading', { name: /Forklift/ })).toBeInTheDocument;
```

Comment:
it's weird that APP_TITLE wouldn't work here, these tests would fail if you have `BRAND_TYPE=RedHat` in your .env file. I guess that's fine though, if it's a problem we can fix it later.